src/main/scala/cromwell/binding/types/WdlType.scala

```diff
@@ -57,11 +59,30 @@ trait WdlType {
 object WdlType {
   val parser = new WdlParser()
 
-  private lazy val wdlTypes = Seq(
-    WdlBooleanType, WdlExpressionType, WdlFileType, WdlFloatType,
-    WdlIntegerType, WdlNamespaceType, WdlObjectType, WdlStringType
+  /* This is in the order of coercion from non-wdl types */
+  lazy val wdlTypes: Seq[WdlType] = Seq(
```

Comment:
Why is this lazy?

I'm not sure if laziness escapes the idiom of static constants & capitalization, e.g. `WdlTypes`, but IMO no